packages/editor/src/components/document-bar/index.js

```diff
@@ -32,7 +32,7 @@ const TYPE_LABELS = {
 	// translators: 1: Pattern title.
 	wp_pattern: __( 'Editing pattern: %s' ),
 	// translators: 1: Navigation menu title.
-	wp_navigation: __( 'Editing navigation menu: %s' ),
+	wp_navigation: __( 'Editing Navigation Menu: %s' ),
```

Comment:
This seems inconsistent with capitalisation of other entities in this panel. I think we should leave it as it is.